src/metabase/lib_be/task/backfill_entity_ids.clj

```diff
@@ -0,0 +1,141 @@
+(ns metabase.lib-be.task.backfill-entity-ids
+  (:require
+   [clojurewerkz.quartzite.conversion :as conversion]
+   [clojurewerkz.quartzite.jobs :as jobs]
+   [clojurewerkz.quartzite.schedule.simple :as simple]
+   [clojurewerkz.quartzite.triggers :as triggers]
+   [metabase.models.serialization :as serdes]
+   [metabase.models.setting :refer [defsetting]]
+   [metabase.task :as task]
+   [metabase.util.i18n :refer [deferred-tru]]
+   [metabase.util.log :as log]
+   [toucan2.core :as t2]))
+
+(set! *warn-on-reflection* true)
+
+(def ^:dynamic *batch-size*
+  "The number of records the backfill entity ids job will process at once"
+  50)
+(def min-repeat-ms
+  "The minimum acceptable repeat rate for the backfill entity ids job"
+  1000)
+(defsetting backfill-entity-ids-repeat-ms
+  (deferred-tru "Frequency for running backfill entity ids job in ms.  Any value below 1000 will disable the job entirely.")
```

Comment:
I think this is a bit surprising. I would have expected `0` to disable it, and eg. `500` to result in a `(log/infof "Backfill entity id interval of %dms is too short; using %dms" repeat-ms min-repeat-ms)` and rounding up.